injectTranslations.py

```diff
@@ -11,47 +11,53 @@
 import sys
 import collections
 
+
 def loadLanguageDict(filename):
     codeToLang = collections.OrderedDict()
-    file = open(filename)
-    for line in file:
-        regex = re.compile("([a-z][a-z]|[a-z][a-z]-[A-Z][A-Z]|[a-z][a-z][a-z]|[a-z][a-z]-[A-Z][A-Z]-.*?)\\s*=\\s*(.*)")  # e.g. "de", "de-DE", "ast", "ca-ES-valencia"
-        match = regex.match(line)
-        if match:
-            codeToLang[match.group(1)] = match.group(2)
-    return codeToLang
+    with open(filename) as File:
```

Comment:
`File` with a capital `F` is a PEP 8 violation.  A variable name I often use here is `handle`, as `open` returns an object that is like a "file handle" in lower layers.

Also, you do not need to add another indent level here.

```python
    with open(filename) as handle:
        lines = handle.readlines()
    for line in lines:
        ....
```

While the above comes at a small cost of reading the entire file into memory before processing, that doesnt look like it will be a problem with this algorithm, as the files are all only ~500 lines.

A benefit is that the file handle is released early, and any file reading or encoding exception handling can be done first, or the backtrace occurs in a simple block, which will make it easier for a newbie to debug.

But the most important benefit is that you dont need to disrupt the other lines of code in this function by indenting them.  The history of those lines (from `git blame`) is not modified.